templates/data/mongodb/what-is-mongodb.html

```diff
@@ -0,0 +1,711 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1hET-ylqXq96mOfj4hovqilX5Nl0J8FLw1gkRMJkIjp8/edit
+{% endblock meta_copydoc %}
+
+{% block title %}What is MongoDB?{% endblock %}
+
+{% block meta_description %}
+  MongoDB is a NoSQL database management application. Organisations use MongoDB because it aligns with modern application development needs, offering flexibility, scalability and high performance. It has a rich set of features that support complex and evolving business requirements.
+{% endblock %}
+
+{% block canonical_url %}{{ request.host_url }}data/mongodb/what-is-opensearch{% endblock %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-section--hero">
+    <div class="p-section">
+      <div class="row--50-50">
+        <div class="col">
+          <div class="p-section--shallow">
+            <h1>
+              What is MongoDB<sup>&reg;</sup>?
+            </h1>
+            <p>
+              MongoDB is a NoSQL database management application. Organisations use MongoDB because it aligns with modern application development needs, offering flexibility, scalability and high performance. It has a rich set of features that support complex and evolving business requirements. Its ability to handle diverse data types, ease of use, cost-effectiveness, and robust security and support services make it a preferred choice for many large organizations.
+            </p>
+          </div>
+          <hr class="is-muted" />
+          <p>
+            <a href="https://ubuntu.com/engage/mongodb-enterprise-data-management">MongoDB<sup>&reg;</sup> for enterprise data management&nbsp;&rsaquo;</a>
+          </p>
+        </div>
+        <div class="col">
+          <div class="u-hide--small">
+            {{ image(url="https://assets.ubuntu.com/v1/e7e981d2-search-image.png",
+                        alt="",
+                        width="957",
+                        height="600",
+                        hi_def=True,
+                        loading="lazy") | safe
```

Comment:
```suggestion
                        loading="auto") | safe
```
As this will be seen on page load, we can set it to auto